cpont/README.md

```diff
@@ -2,9 +2,13 @@
 
 This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Critical Path](https://c-path.org/) Ontology (CPONT).
 
-The w3id.org/cpont redirect is used to direct URIs for terms, documentation, and repositories related to CPONT.
+The `w3id.org/cpont` redirect is used to direct URIs for terms, documentation, and repositories related to CPONT.
 
-The top-level redirects to the primary GitLab repository for the ontology project, [here](https://gitlab.c-path.org/c-pathontology/critical-path-ontology/).
+The `w3id.org/cpont/mappings` redirect is used to direct URIs for sssom mapping files in the CPONT Mapping Commons.
```

Comment:
```suggestion
The `w3id.org/cpont/mappings` redirect is used to direct URIs for some mapping files in the CPONT Mapping Commons.
```

Reply:
"sssom" isn't a typo.  Would adding the following hyperlink help clarify this?
[sssom](https://mapping-commons.github.io/sssom/)